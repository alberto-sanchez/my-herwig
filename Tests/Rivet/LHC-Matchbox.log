>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>> Toolkit for HEP Event Generation - ThePEG 72c02dbb1f8d (release-2-1) <<<<
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

>> Herwig dd9178836fc4 (herwig-7-1) / ThePEG 72c02dbb1f8d (release-2-1)

/Herwig/Cuts/Cuts:
MHat  = 0 .. 13000 GeV
Scale = 2 .. 1e+12 GeV2
YHat  = -100 .. 100
X1    = 1e-05 .. 1
X2    = 1e-05 .. 1

IdentifiedParticleCut 'PhotonCut' matching 'Photon' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'LeptonCut' matching 'Lepton' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'TopQuarkCut' matching 'Top' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'BottomQuarkCut' matching 'Bottom' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'WBosonCut' matching 'WBoson' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'ZBosonCut' matching 'ZBoson' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'HiggsBosonCut' matching 'HiggsBoson' within:
pt  = 0 .. 1e+06 GeV
IdentifiedParticleCut 'ChargedLeptonCut' matching 'ChargedLepton' within:
pt  = 0 .. 1e+06 GeV
/Herwig/Cuts/LeptonPairMassCut
matching distances between: 'Lepton' and 'Lepton':
M = 0 .. 1e+06 GeV
same flavour only = false 
opposite sign only = false 

/Herwig/Cuts/ChargedLeptonPairMassCut
matching distances between: 'ChargedLepton' and 'ChargedLepton':
M = 60 .. 120 GeV
same flavour only = false 
opposite sign only = false 

/Herwig/Cuts/LeptonDeltaRCut
matching distances between: 'Lepton' and 'Lepton':
DeltaRMin = 0 
DeltaRMax = 100 
DeltaPhiMin = 0 
DeltaPhiMax = 6.28319 
DeltaYMin = 0 
DeltaYMax = 100 

/Herwig/Cuts/ChargedLeptonDeltaRCut
matching distances between: 'ChargedLepton' and 'ChargedLepton':
DeltaRMin = 0 
DeltaRMax = 100 
DeltaPhiMin = 0 
DeltaPhiMax = 6.28319 
DeltaYMin = 0 
DeltaYMax = 100 

/Herwig/Cuts/PhotonIsolationCut matching unresolved particles from 'MatchboxJetMatcher':
DeltaZero = 0 
Exponent n = 1 
Efficiency = 0 
Cut Type = 1 

MissingPtCut 'MissingPtCut' matching 'Neutrino' within:
ptMiss  = 0 .. 1e+06 GeV
******************************************************************************
Event number 1 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    17.221    17.221     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -229.377   229.377     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000  -212.156   246.598   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    17.221    17.221     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -229.377   229.377     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000  -212.156   246.598   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6482.779  6482.779     0.937
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6270.623  6270.623     0.922
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 2 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    46.447    46.447     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000   -85.045    85.045     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000   -38.598   131.493   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    46.447    46.447     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000   -85.045    85.045     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000   -38.598   131.493   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6453.553  6453.553     0.935
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6414.955  6414.955     0.932
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 3 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    24.603    24.603     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -160.553   160.553     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000  -135.950   185.156   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    24.603    24.603     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -160.553   160.553     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000  -135.950   185.156   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6475.397  6475.397     0.936
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6339.447  6339.447     0.927
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 4 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     6.044     6.044     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -653.518   653.518     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000  -647.474   659.562   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     6.044     6.044     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -653.518   653.518     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000  -647.474   659.562   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6493.956  6493.956     0.938
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -5846.482  5846.482     0.890
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 5 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000   185.729   185.729     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000   -21.268    21.268     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000   164.461   206.997   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000   185.729   185.729     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000   -21.268    21.268     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000   164.461   206.997   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6314.271  6314.271     0.925
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6478.732  6478.732     0.937
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 6 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     1.997     1.997     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000 -1978.449  1978.449     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000 -1976.452  1980.446   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     1.997     1.997     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000 -1978.449  1978.449     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000 -1976.452  1980.446   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6498.003  6498.003     0.938
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -4521.551  4521.551     0.783
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 7 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000   507.307   507.307     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000    -7.786     7.786     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000   499.520   515.093   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000   507.307   507.307     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000    -7.786     7.786     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000   499.520   515.093   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  5992.693  5992.693     0.901
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6492.213  6492.214     0.938
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 8 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     6.782     6.782     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -582.406   582.406     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000  -575.624   589.189   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000     6.782     6.782     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -582.406   582.406     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000  -575.624   589.189   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6493.218  6493.218     0.938
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -5917.594  5917.594     0.895
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
******************************************************************************
Event number 9 (id: LHC-Matchbox) performed by EventHandler
==============================================================================

--- Colliding particles:
  1        p+    2212  (3,6)   
                                 0.000     0.000  6500.000  6500.000     0.938
  2        p+    2212  (4,7)   
                                 0.000     0.000 -6500.000  6500.000     0.938
------------------------------------------------------------------------------
Primary sub-process performed by ME[g,g->h0].BornVirtual
--- incoming:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    25.860    25.860     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -152.752   152.752     0.000
--- outgoing:
  5        h0      25 [3,4]    
                                 0.000     0.000  -126.892   178.611   125.700
------------------------------------------------------------------------------
Step 1
--- intermediates:
  3         g      21 [1] (5)   {+1,-2}
                                 0.000     0.000    25.860    25.860     0.000
  4         g      21 [2] (5)   {+2,-1}
                                 0.000     0.000  -152.752   152.752     0.000
--- final:
  5        h0      25 [3,4]    
                                 0.000     0.000  -126.892   178.611   125.700
  6    Rem:p+      82 [1]   7>>7 {+2,-1}
                                 0.000     0.000  6474.140  6474.140     0.936
  7    Rem:p+      82 [2]   6>>6 {+1,-2}
                                 0.000     0.000 -6347.248  6347.248     0.927
------------------------------------------------------------------------------
     Sum of momenta:                 0         0         0   1.3e+04   1.3e+04
No exceptions reported in this run.
