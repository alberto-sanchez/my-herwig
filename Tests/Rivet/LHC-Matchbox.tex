\documentclass{article}
\usepackage{graphics}
\begin{document}
\appendix
\section[xxx]{\textsc{ThePEG} version ThePEG 72c02dbb1f8d (release-2-1) \cite{ThePEG} Run Information}
Run name: \textbf{LHC-Matchbox}:\\
This run was generated using Herwig~\cite{Bahr:2008pv} and the following models:
\begin{itemize}
\item NLO QCD corrections have been calculated using Matchbox \cite{Platzer:2011bc}, \cite{Matchbox:2015}
\item The weak decay of baryons containing a heavy quark used form factors from \cite{Cheng:1995fe,Cheng:1996cs}.
\end{itemize}

\begin{thebibliography}{99}
\bibitem{ThePEG} L.~L\"onnblad, Comput.~Phys.~Commun.\ {\bf 118} (1999) 213.
\bibitem{Bahr:2008pv}
  M.~Bahr {\it et al.},
  ``Herwig Physics and Manual,''
  Eur.\ Phys.\ J.\  C {\bf 58} (2008) 639
  [arXiv:0803.0883 [hep-ph]].
  %%CITATION = EPHJA,C58,639;%%

%\cite{Platzer:2011bc}
\bibitem{Platzer:2011bc}
S.~Platzer and S.~Gieseke,
``Dipole Showers and Automated NLO Matching in Herwig,''
arXiv:1109.6256 [hep-ph].
%%CITATION = ARXIV:1109.6256;%%
%\cite{Matchbox:2015}
\bibitem{Matchbox:2015}
Herwig collaboration,
``Precision LHC Event Generation with Herwig,''
in preparation.
%\cite{Cheng:1995fe}
\bibitem{Cheng:1995fe}
  H.~Y.~Cheng and B.~Tseng,
  %``1/M corrections to baryonic form-factors in the quark model,''
  Phys.\ Rev.\  D {\bf 53} (1996) 1457
  [Erratum-ibid.\  D {\bf 55} (1997) 1697]
  [arXiv:hep-ph/9502391].
  %%CITATION = PHRVA,D53,1457;%%
%\cite{Cheng:1996cs}
\bibitem{Cheng:1996cs}
  H.~Y.~Cheng,
  %``Nonleptonic weak decays of bottom baryons,''
  Phys.\ Rev.\  D {\bf 56} (1997) 2799
  [arXiv:hep-ph/9612223].
  %%CITATION = PHRVA,D56,2799;%%

\end{thebibliography}
\end{document}
