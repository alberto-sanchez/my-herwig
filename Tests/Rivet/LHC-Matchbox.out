>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>> Toolkit for HEP Event Generation - ThePEG 72c02dbb1f8d (release-2-1) <<<<
<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<

>> Herwig dd9178836fc4 (herwig-7-1) / ThePEG 72c02dbb1f8d (release-2-1)

==============================================================================
Statistics for event handler 'EventHandler':
                                       generated    number of    Cross-section
                                          events     attempts             (nb)
==============================================================================
Total (from   weighted events): including vetoed events            32.7(3)e-03
==============================================================================
Total (from unweighted events):            10000        10000      32.7(3)e-03
==============================================================================
Per matrix element breakdown:
ME[g,g->h0].BornVirtual                    10000        10000      32.7(3)e-03
==============================================================================
